x/incentive/keeper/refund_tx_decorator.go

```diff
@@ -18,6 +29,29 @@ func NewRefundTxDecorator(k *Keeper) *RefundTxDecorator {
 	}
 }
 
+// Makes sure there's no fee granter assigned in refundable txs
+// so there's no need to restore allowances
+func (d *RefundTxDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simulate bool, next sdk.AnteHandler) (sdk.Context, error) {
```

Comment:
maybe we can some docs in `x/incentive` module that explain that it is not possible to use fee grants for refundable transactions ?

Reply:
updated the module's README. LMK if we want to include this somewhere else